Lib/glyphsLib/builder/kerning.py

```diff
@@ -54,7 +54,12 @@ def _to_ufo_kerning(self, ufo, kerning_data):
             ufo.kerning[left, right] = kerning_val
 
     seen = {}
-    for classname, glyph, is_left_class in reversed(class_glyph_pairs):
```

Comment:
why was it `reversed` before?

Reply:
Blaming gives this: https://github.com/googlei18n/glyphsLib/commit/439676094ce3fc155ffdc4228447aa09695a25c0

Reply:
> kerning conflicts are checked and resolved in reverse order of how they appear in Glyphs source.

hm...  but isn't `kerning_data.items()` an unsorted collection?

Reply:
no, it's an OrderedDict apparently